easybuild/easyconfigs/e/expat/expat-2.2.1-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'expat'
+version = '2.2.1'
```

Comment:
@JackPerdue We already have an easyconfig for `expat` 2.2.5, including this old version doesn't make sense?